modules/web/src/assets/css/global/_main.scss

```
@@ -719,6 +719,95 @@ km-cni-version {
   pointer-events: none;
 }
 
+// style for valueChanged directive
+
+.value-changed {
```

Since its our global class.
```suggestion
.km-value-changed {
```